doc.html

```diff
@@ -0,0 +1,111 @@
+<html>
+<head>
+    <style type="text/css">
+        body { font-family: courier; font-size:12px; margin-left: 42px; width: 465px; white-space: pre-line; }
+        h2 { margin: 10px 0 0 -22px; font-size:12px; padding: 0 !important }
+        table { font-size: 10px; margin-left: -22px; width: 480px; overflow-wrap: break-work; }
+        table td { overflow-wrap: break-work; }
+        div.header { padding: 0; margin: 0 0 0 -22px; white-space: pre-wrap; }
+    </style>
+</head>
+<body>
+
+<div class="header">
+ATLAS Service Template Specification
+Name                                                    feed_data
+Version                                                     0.0.3
+Author(s)             ? (MmmooOgle), J. V&ouml;lker (fodjan GmbH)
```

Comment:
Please indicate proper MmmooOgle contributor or just put it MmmmooOgle generically (no ?, no parenthensis)